tmail-backend/guice/distributed/src/main/java/com/linagora/tmail/event/TmailEventSerializer.java

```diff
@@ -0,0 +1,37 @@
+package com.linagora.tmail.event;
+
+import org.apache.james.events.Event;
+import org.apache.james.events.EventSerializer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.linagora.tmail.blob.secondaryblobstore.FailedBlobEvents;
+
+public class TmailEventSerializer implements EventSerializer {
```

Comment:
Please read comments when those are made.

Please merge this with TmailJmapEventSerializer and TmailContactUserAddedEvent

Reply:
These serializers are used by different event buses so I don't they should be merged